### ISSUE TITLE
How can I get the distributed transaction rate?

### ISSUE THREAD

**Issue by luyiqu** [pingcap/tidb#26264]

**Comment by ichn-hu** [comment]
do you mean TPS? we have a TiKV panel in grafana, see https://docs.pingcap.com/tidb/stable/deploy-monitoring-services (or [Chinese version](https://docs.pingcap.com/zh/tidb/stable/grafana-monitor-best-practices))

**Comment by luyiqu** [comment]
> do you mean TPS? we have a TiKV panel in grafana, see https://docs.pingcap.com/tidb/stable/deploy-monitoring-services (or [Chinese version](https://docs.pingcap.com/zh/tidb/stable/grafana-monitor-best-practices))

No, if tikv is deployed on three nodes ,  how can I know a transaction access the number of nodes?
